### DISCUSSION TITLE
Feature request: Select multiple

### DISCUSSION THREAD

**Discussion by tumeboy** [beercss/beercss#390] [Ideas]
I have been thrilled to use beercss lately! I've managed to find pretty much everything I've wanted but the lack of support for native multiselect hinders at times. Especially on mobile browsers, the native implementation works quite well IMO. 

Could it be possible to implement this in the future? I am also welcome to suggestions if anyone knew if the select multiple could be made working right away.

`<select name="color" id="id_color" multiple>`

**Comment by ns8482e** [comment]
I am looking for solution for same- is the alternative solution for this?

**Comment by ns8482e** [comment]
If anyone interested - here's the workaround https://codepen.io/ns8482e/full/KwPgjLO.

**Comment by leonardorafael** [comment]
Unfortunable the Material Design System doesn't provide a multiple select options. They have another ways of selections like: 

MD3: https://m3.material.io/foundations/interaction/selection
MD2: https://m2.material.io/components/selection-controls
Stackoverflow: https://stackoverflow.com/questions/63845044/how-to-implement-multi-select-in-material-components-for-the-web

As you see, we don't have a clear way to style a multiple select with material design system today.

**Reply by ns8482e** [reply]
It seems `<select multiple>`  is discouraged in MD

**Reply by ns8482e** [reply]
> It seems `<select multiple>` is discouraged in MD

I was wrong - MD3 do support `<select>` element - in MD3 `exposed dropdown menu` combined into menus.

https://m3.material.io/components/menus/overview#b39b2819-a579-4de1-bca2-8d7dbd3b332c 

@leonardorafael  it would be nice to support `<select>` tag with multiple, where select could open menu and each click would change check/uncheck on menu item
